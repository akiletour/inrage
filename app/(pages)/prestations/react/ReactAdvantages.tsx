import NextJsAdvantageCard from '@component/items/NextJsAdvantageCard'

export default function ReactAdvantages() {
  const advantages = [
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2L2 7v10c0 5.55 3.84 9.739 9 11 5.16-1.261 9-5.45 9-11V7l-10-5z" />
        </svg>
      ),
      title: 'Performance Optimisée',
      description:
        'Next.js offre des optimisations automatiques avec le SSR, SSG et le code splitting pour des temps de chargement ultra-rapides.',
      highlight: 'Core Web Vitals',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z" />
        </svg>
      ),
      title: 'SEO Natif',
      description:
        'Le rendu côté serveur et la génération statique garantissent une indexation parfaite par les moteurs de recherche.',
      highlight: 'Google Ready',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96z" />
        </svg>
      ),
      title: 'Scalabilité',
      description:
        'Architecture composants modulaire permettant de faire évoluer facilement vos applications avec votre business.',
      highlight: 'Enterprise Ready',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2C13.1 2 14 2.9 14 4C14 5.1 13.1 6 12 6C10.9 6 10 5.1 10 4C10 2.9 10.9 2 12 2ZM21 9V7L15 1L13.5 2.5L16.17 5.24C15.24 5.9 15 6.32 15 7V9H21ZM3 23H5L14 13.91V22H8V19H6V22H0V19C0 18.46 0.22 17.94 0.61 17.55L7.05 11.1L7.55 12.1H9V14C9 14.55 8.78 15.05 8.39 15.44L6 17.83V19H8V16.5L9.5 15H11.24L3 23Z" />
        </svg>
      ),
      title: 'Développement Moderne',
      description:
        'TypeScript intégré, Hot Module Replacement et outils de développement avancés pour une productivité maximale.',
      highlight: 'Developer Experience',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2l3.09 6.26L22 9.27l-5 4.87 1.18 6.88L12 17.77l-6.18 3.25L7 14.14 2 9.27l6.91-1.01L12 2z" />
        </svg>
      ),
      title: 'Écosystème Riche',
      description:
        'Vaste communauté, librairies maintenues et intégrations tierces pour accélérer le développement.',
      highlight: 'Open Source',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M20 4H4C2.9 4 2 4.9 2 6v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8h16v10z" />
        </svg>
      ),
      title: 'Responsive Design',
      description:
        "Composants adaptatifs par défaut et optimisations mobiles intégrées pour toutes les tailles d'écran.",
      highlight: 'Mobile First',
    },
  ]

  return (
    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 mt-16">
      {advantages.map((advantage, index) => (
        <div
          key={index}
          className="animate-in slide-in-from-bottom-4 duration-700"
          style={{ animationDelay: `${index * 150}ms` }}
        >
          <NextJsAdvantageCard
            icon={advantage.icon}
            title={advantage.title}
            description={advantage.description}
            highlight={advantage.highlight}
          />
        </div>
      ))}
    </div>
  )
}
