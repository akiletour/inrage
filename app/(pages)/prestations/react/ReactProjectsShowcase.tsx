import ReactProjectShowcase from '@component/items/ReactProjectShowcase'

export default function ReactProjectsShowcase() {
  const projects = [
    {
      title: 'Portfolio inRage (Ce site)',
      description: 'Portfolio personnel développé entièrement avec Next.js 15, TypeScript et Tailwind CSS. Architecture moderne avec App Router, optimisations SEO et performance.',
      technologies: ['Next.js 15', 'TypeScript', 'Tailwind CSS', 'WordPress Headless', 'GraphQL'],
      metrics: [
        { label: 'Performance', value: '95/100' },
        { label: 'SEO', value: '98/100' },
        { label: 'Accessibility', value: '94/100' }
      ],
      githubUrl: 'https://github.com/akiletour/inrage',
      codeIcon: (
        <svg className="w-4 h-4 text-white" fill="currentColor" viewBox="0 0 24 24">
          <path d="M12 0c-6.626 0-12 5.373-12 12 0 5.302 3.438 9.8 8.207 11.387.599.111.793-.261.793-.577v-2.234c-3.338.726-4.033-1.416-4.033-1.416-.546-1.387-1.333-1.756-1.333-1.756-1.089-.745.083-.729.083-.729 1.205.084 1.839 1.237 1.839 1.237 1.07 1.834 2.807 1.304 3.492.997.107-.775.418-1.305.762-1.604-2.665-.305-5.467-1.334-5.467-5.931 0-1.311.469-2.381 1.236-3.221-.124-.303-.535-1.524.117-3.176 0 0 1.008-.322 3.301 1.23.957-.266 1.983-.399 3.003-.404 1.02.005 2.047.138 3.006.404 2.291-1.552 3.297-1.23 3.297-1.30.653 1.653.242 2.874.118 3.176.77.84 1.235 1.911 1.235 3.221 0 4.609-2.807 5.624-5.479 5.921.43.372.823 1.102.823 2.222v3.293c0 .319.192.694.801.576 4.765-1.589 8.199-6.086 8.199-11.386 0-6.627-5.373-12-12-12z"/>
        </svg>
      )
    },
    {
      title: 'Dashboard E-commerce',
      description: 'Interface d\'administration pour plateforme e-commerce avec gestion des commandes, statistiques en temps réel et inventaire. Développé avec React et Chart.js.',
      technologies: ['React 18', 'TypeScript', 'Chart.js', 'Material-UI', 'Redux Toolkit'],
      metrics: [
        { label: 'Utilisateurs', value: '500+' },
        { label: 'Commandes/jour', value: '2.5k' },
        { label: 'Uptime', value: '99.9%' }
      ]
    },
    {
      title: 'Application SaaS Marketing',
      description: 'Plateforme SaaS pour gestion de campagnes marketing avec analytics avancés, A/B testing et intégrations multiples. Architecture micro-frontends.',
      technologies: ['Next.js 14', 'React Query', 'Prisma', 'tRPC', 'Stripe'],
      metrics: [
        { label: 'Clients actifs', value: '1.2k' },
        { label: 'API Calls/min', value: '10k' },
        { label: 'Conversion', value: '+25%' }
      ]
    },
    {
      title: 'App Mobile React Native',
      description: 'Application mobile cross-platform pour la gestion de tâches collaboratives avec synchronisation temps réel et notifications push.',
      technologies: ['React Native', 'Expo', 'Firebase', 'TypeScript', 'Zustand'],
      metrics: [
        { label: 'Downloads', value: '15k+' },
        { label: 'Rating', value: '4.8/5' },
        { label: 'Retention', value: '78%' }
      ]
    },
    {
      title: 'Site E-commerce Mode',
      description: 'Boutique en ligne haute performance avec catalogue produits dynamique, panier avancé et checkout optimisé. Intégration Stripe et stock management.',
      technologies: ['Next.js', 'Shopify API', 'Framer Motion', 'Stripe', 'Vercel'],
      metrics: [
        { label: 'Conversion', value: '3.2%' },
        { label: 'Page Speed', value: '0.8s' },
        { label: 'Mobile Score', value: '96/100' }
      ]
    },
    {
      title: 'Plateforme Formation',
      description: 'LMS (Learning Management System) avec player vidéo personnalisé, progression tracking et certification automatique. Interface élève et formateur.',
      technologies: ['React', 'Node.js', 'Video.js', 'PostgreSQL', 'AWS S3'],
      metrics: [
        { label: 'Étudiants', value: '3.5k' },
        { label: 'Cours', value: '250+' },
        { label: 'Complétion', value: '89%' }
      ]
    }
  ]

  return (
    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 mt-8">
      {projects.map((project, index) => (
        <div
          key={index}
          className="animate-in slide-in-from-bottom-6 duration-700"
          style={{ animationDelay: `${index * 120}ms` }}
        >
          <ReactProjectShowcase
            title={project.title}
            description={project.description}
            technologies={project.technologies}
            metrics={project.metrics}
            codeIcon={project.codeIcon}
            githubUrl={project.githubUrl}
          />
        </div>
      ))}
    </div>
  )
}