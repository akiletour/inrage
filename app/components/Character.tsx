import style from './Character.module.css'

export default function Character() {
  return (
    <div>
      <div className={style.newcharacter404}>
        <div className={style.words}>
          <div className={style.word}>&lt;?php</div>
          <div className={style.word}>&lt;div&gt;</div>
        </div>
        <div className={style.words}>
          <div className={style['word-right']}>&lt;html&gt;</div>
          <div className={style['word-right']}>body{}</div>
        </div>
        <div className={style.chair404}></div>
        <div className={style.leftshoe404}></div>
        <div className={style.rightshoe404}></div>
        <div className={style.legs404}></div>
        <div className={style.torso404}>
          <div className={style.body404}></div>
          <div className={style.leftarm404}></div>
          <div className={style.rightarm404}></div>
          <div className={style.head404}>
            <div className={style.eyes404}></div>
          </div>
        </div>
        <div className={style.laptop404}></div>
      </div>
    </div>
  )
}
