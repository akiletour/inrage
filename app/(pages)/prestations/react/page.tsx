import Image from 'next/image'

import ContactForm from '@component/ContactForm'
import Diagonal from '@component/Diagonal'
import Layout from '@component/Layout'
import SectionTitle from '@component/SectionTitle'
import ReactServicesGrid from './ReactServicesGrid'
import ReactAdvantages from './ReactAdvantages'
import ReactProjectsShowcase from './ReactProjectsShowcase'
import { RouteLink, getCanonicalUrl } from '@lib/router'

export const metadata = {
  title: 'Développeur React Freelance La Rochelle | Expert Next.js - inRage',
  description:
    "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes. Spécialisé TypeScript, SSR, e-commerce. Devis gratuit.",
  keywords: [
    'développeur react freelance la rochelle',
    'expert nextjs la rochelle',
    'création application react',
    'développement spa react',
    'freelance typescript la rochelle',
    'développeur frontend react france',
  ],
  alternates: {
    canonical: getCanonicalUrl(RouteLink.prestationReact),
  },
  openGraph: {
    title: 'Développeur React Freelance La Rochelle | Expert Next.js',
    description:
      "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes pour votre business.",
    url: getCanonicalUrl(RouteLink.prestationReact),
    siteName: 'inRage',
    locale: 'fr_FR',
    type: 'website',
    images: [
      {
        url: 'https://www.inrage.fr/images/react-nextjs-development.png',
        width: 697,
        height: 736,
        alt: 'Développement React Next.js applications modernes La Rochelle',
      },
    ],
  },
  twitter: {
    card: 'summary_large_image',
    title: 'Développeur React Freelance La Rochelle | Expert Next.js',
    description:
      "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes.",
    images: ['https://www.inrage.fr/images/react-nextjs-development.png'],
  },
  robots: {
    index: true,
    follow: true,
    googleBot: {
      index: true,
      follow: true,
      'max-video-preview': -1,
      'max-image-preview': 'large',
      'max-snippet': -1,
    },
  },
}

export default function PrestationReact() {
  const reactServiceSchema = {
    '@context': 'https://schema.org',
    '@type': 'Service',
    name: 'Développement React et Next.js',
    description:
      "Service de développement d'applications web modernes avec React, Next.js et TypeScript par un développeur freelance expert basé à La Rochelle",
    provider: {
      '@type': 'Person',
      name: 'Pascal Gault',
      jobTitle: 'Développeur React Freelance',
      address: {
        '@type': 'PostalAddress',
        addressLocality: 'La Rochelle',
        addressRegion: 'Nouvelle-Aquitaine',
        postalCode: '17000',
        addressCountry: 'FR',
      },
      url: 'https://www.inrage.fr',
      sameAs: [
        'https://www.linkedin.com/in/akiletour/',
        'https://github.com/akiletour/inrage',
      ],
    },
    serviceType: 'Développement Web',
    category: "Développement d'applications",
    offers: {
      '@type': 'Offer',
      description: "Développement d'applications React et Next.js",
      price: 'Sur devis',
      priceCurrency: 'EUR',
      availability: 'https://schema.org/InStock',
    },
    areaServed: {
      '@type': 'Place',
      name: 'France',
      containedInPlace: {
        '@type': 'Place',
        name: 'La Rochelle',
      },
    },
    hasOfferCatalog: {
      '@type': 'OfferCatalog',
      name: 'Services React & Next.js',
      itemListElement: [
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: "Création d'applications React",
            description:
              "Développement d'applications single-page modernes avec React",
          },
        },
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: 'Développement Next.js',
            description:
              'Applications Next.js avec SSR/SSG pour performances optimales',
          },
        },
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: 'Migration vers React',
            description:
              "Migration d'applications existantes vers l'écosystème React",
          },
        },
      ],
    },
  }

  const breadcrumbSchema = {
    '@context': 'https://schema.org',
    '@type': 'BreadcrumbList',
    itemListElement: [
      {
        '@type': 'ListItem',
        position: 1,
        name: 'Accueil',
        item: 'https://www.inrage.fr',
      },
      {
        '@type': 'ListItem',
        position: 2,
        name: 'Prestations',
        item: 'https://www.inrage.fr/prestations',
      },
      {
        '@type': 'ListItem',
        position: 3,
        name: 'Développement React & Next.js',
        item: 'https://www.inrage.fr/prestations/react',
      },
    ],
  }

  const faqSchema = {
    '@context': 'https://schema.org',
    '@type': 'FAQPage',
    mainEntity: [
      {
        '@type': 'Question',
        name: 'Pourquoi choisir un développeur React freelance à La Rochelle ?',
        acceptedAnswer: {
          '@type': 'Answer',
          text: 'Un développeur React freelance à La Rochelle offre une expertise locale avec une connaissance approfondie du marché français. Je propose un accompagnement personnalisé, des délais flexibles et une communication directe pour vos projets React et Next.js.',
        },
      },
      {
        '@type': 'Question',
        name: 'Quels sont les avantages de React et Next.js pour mon projet ?',
        acceptedAnswer: {
          '@type': 'Answer',
          text: 'React et Next.js offrent des performances excellentes, un SEO optimisé avec le rendu côté serveur, une évolutivité maximale et une expérience utilisateur fluide. Ces technologies modernes garantissent la pérennité de votre application web.',
        },
      },
      {
        '@type': 'Question',
        name: "Combien coûte le développement d'une application React ?",
        acceptedAnswer: {
          '@type': 'Answer',
          text: "Le coût dépend de la complexité du projet. Une application React simple commence à partir de 3000€, tandis qu'une solution complète avec Next.js peut varier de 5000€ à 15000€. Je propose toujours un devis gratuit personnalisé.",
        },
      },
      {
        '@type': 'Question',
        name: "Quels types d'applications React développez-vous ?",
        acceptedAnswer: {
          '@type': 'Answer',
          text: "Je développe tous types d'applications : SPA (Single Page Applications), sites e-commerce, dashboards d'administration, plateformes SaaS, applications métier et sites vitrines modernes avec React et Next.js.",
        },
      },
    ],
  }

  return (
    <>
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(reactServiceSchema) }}
      />
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(breadcrumbSchema) }}
      />
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(faqSchema) }}
      />
      <Layout
        breadcrumbs={[
          {
            link: RouteLink.prestations,
            title: 'Prestations sous WordPress, Prestashop et Symfony',
          },
        ]}
        title="Développement React & Next.js"
      >
        <div className="flex max-lg:flex-col relative container gap-16 *:flex-1">
          <Image
            className="h-full! object-cover drop-shadow-2xl"
            src="/images/react-nextjs-development.webp"
            width={697}
            priority
            height={736}
            alt="Développeur React freelance La Rochelle - Création applications Next.js modernes"
            style={{
              maxWidth: '100%',
              height: 'auto',
            }}
          />
          <div className=" relative flex items-center">
            <div className="space-y-12">
              <div className="space-y-8">
                <h1 className="font-bold text-white text-2xl md:text-4xl lg:text-5xl leading-tight animate-in slide-in-from-bottom-4 duration-700">
                  Développeur React freelance à
                  <span className="text-orange"> La Rochelle</span>
                </h1>

                <div className="text-gray-light font-medium text-lg md:text-xl leading-relaxed animate-in slide-in-from-bottom-6 duration-700 delay-150">
                  <p>
                    Expert React et Next.js freelance basé à La Rochelle,
                    spécialisé dans la création d&apos;applications web
                    performantes avec React, Next.js et TypeScript.
                  </p>
                </div>
              </div>

              <div className="space-y-8 text-gray-light/90 leading-relaxed animate-in slide-in-from-bottom-8 duration-700 delay-300">
                <p>
                  Développeur React freelance à La Rochelle, je conçois et
                  développe des applications web modernes, réactives et
                  scalables en utilisant l&apos;écosystème React. De
                  l&apos;application single-page au site e-commerce complexe, en
                  passant par les dashboards et les plateformes SaaS pour
                  entreprises françaises.
                </p>
                <p>
                  Mon expertise en tant que développeur React freelance La
                  Rochelle couvre React, Next.js, TypeScript, ainsi que
                  l&apos;intégration d&apos;APIs REST et GraphQL pour créer des
                  expériences utilisateur fluides et performantes adaptées au
                  marché français.
                </p>
                <p>
                  Je vous accompagne dans la modernisation de vos outils
                  existants ou la création de nouvelles applications avec les
                  technologies les plus récentes de l&apos;écosystème React.
                </p>
              </div>

              <div className="flex flex-col sm:flex-row gap-8 pt-8 animate-in slide-in-from-bottom-10 duration-700 delay-500">
                <a
                  href="#contact"
                  className="button-primary group inline-flex items-center justify-center px-12 py-6 rounded-lg font-semibold text-white bg-orange hover:bg-orange-dark transform hover:scale-105 transition-all duration-300 shadow-lg hover:shadow-orange/25"
                >
                  Démarrer mon projet
                  <svg
                    className="w-10 h-10 ml-4 transform group-hover:translate-x-2 transition-transform"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                      d="M17 8l4 4m0 0l-4 4m4-4H3"
                    />
                  </svg>
                </a>
                <a
                  href="#services"
                  className="button-secondary inline-flex items-center justify-center px-12 py-6 rounded-lg font-semibold text-orange border-2 border-orange hover:bg-orange hover:text-white transition-all duration-300"
                >
                  Voir mes services
                </a>
              </div>
            </div>
          </div>
        </div>

        <div className="bg-gray-darker mt-20 relative overflow-hidden">
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <Diagonal
            flipX
            flipY
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgClass="fill-gray-dark"
            bgCorner="fill-gray-darker"
          />
          <div className="container text-center relative z-10">
            <div className="md:px-36 text-lg max-w-4xl mx-auto">
              <h2 className="text-white text-3xl md:text-4xl lg:text-5xl font-bold mb-16 leading-tight">
                Développement React à La Rochelle
                <br />
                <span className="text-orange">au service de vos projets</span>
              </h2>

              <div className="grid grid-cols-1 md:grid-cols-3 gap-16 mt-24">
                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0L19.2 12l-4.6-4.6L16 6l6 6-6 6-1.4-1.4zM12 18l2-8h-4l2 8z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Technologies modernes
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    React et Next.js représentent aujourd&apos;hui les
                    technologies de référence pour le développement
                    d&apos;applications web modernes.
                  </p>
                </div>

                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M12 2l3.5 6.5h7L17 14l2.5 7.5L12 18l-7.5 3.5L7 14 1.5 8.5h7L12 2zm0 2.8L9.5 9H4.2l3.3 3.7L6 19l6-3.2L18 19l-1.5-6.3L20.8 9h-5.3L12 4.8z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Approche component-driven
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    J&apos;utilise une approche component-driven pour créer des
                    interfaces réutilisables et maintenables avec une
                    architecture solide.
                  </p>
                </div>

                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-1 17.93c-3.94-.49-7-3.85-7-7.93 0-.62.08-1.21.21-1.79L9 15v1c0 1.1.9 2 2 2v1.93zm6.9-2.54c-.26-.81-1-1.39-1.9-1.39h-1v-3c0-.55-.45-1-1-1H8v-2h2c.55 0 1-.45 1-1V7h2c1.1 0 2-.9 2-2v-.41c2.93 1.19 5 4.06 5 7.41 0 2.08-.8 3.97-2.1 5.39z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Adaptabilité
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    Que ce soit pour un site vitrine, une application e-commerce
                    ou un dashboard, je m&apos;adapte à vos besoins spécifiques.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <Diagonal
            bgClass="fill-gray-dark"
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-gray-darker"
          />
        </div>

        <div
          id="services"
          className="container mt-8 md:mt-0 lg:-mb-20 relative z-10 scroll-mt-40"
        >
          <div className="animate-in slide-in-from-bottom-4 duration-700">
            <SectionTitle
              title={['Mes services', 'React & Next.js']}
              content="De la conception à la mise en production, je vous accompagne dans la création d'applications React performantes et modernes adaptées à vos besoins business."
            />
          </div>

          <ReactServicesGrid />
        </div>

        <Diagonal
          className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
          bgCorner="fill-orange"
          bgClass="fill-gray-darker"
        />

        <div className="bg-gray-darker relative overflow-hidden">
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <div className="container mb-8 md:mb-0 relative z-10">
            <div className="animate-in slide-in-from-bottom-4 duration-700">
              <SectionTitle
                content="Découvrez pourquoi React et Next.js sont les technologies de choix pour vos projets web modernes."
                title="Avantages React & Next.js"
              />
            </div>

            <ReactAdvantages />
          </div>

          <Diagonal
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-orange"
            bgClass="fill-gray-dark"
          />
        </div>

        <div className="container mt-8 md:mt-0">
          <div className="animate-in slide-in-from-bottom-4 duration-700">
            <SectionTitle
              content="Découvrez quelques exemples de réalisations React et Next.js que j'ai développées pour mes clients."
              title="Projets React & Next.js"
            />
          </div>

          <ReactProjectsShowcase />
        </div>

        <Diagonal
          className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
          bgCorner="fill-orange"
          bgClass="fill-gray-darker"
        />

        <div
          id="contact"
          className="bg-gray-darker relative overflow-hidden scroll-mt-40"
        >
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <div className="container relative z-10">
            <div className="animate-in slide-in-from-bottom-4 duration-700">
              <SectionTitle
                content="Prêt à moderniser vos applications web avec React et Next.js ? Contactez-moi pour discuter de votre projet."
                title="Démarrons votre projet"
              />
            </div>

            <div className="animate-in slide-in-from-bottom-6 duration-700 delay-200">
              <ContactForm />
            </div>
          </div>

          <Diagonal
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-orange"
            bgClass="fill-gray-dark"
          />
        </div>
      </Layout>
    </>
  )
}
