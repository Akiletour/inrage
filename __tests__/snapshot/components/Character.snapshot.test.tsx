import { render } from '@testing-library/react';

import Character from '@component/Character';

it('renders Character unchanged', () => {
  const { container } = render(<Character />);
  expect(container).toMatchSnapshot();
});
