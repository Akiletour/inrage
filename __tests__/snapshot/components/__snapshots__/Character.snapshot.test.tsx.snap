// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Character unchanged 1`] = `
<div>
  <div>
    <div
      class="newcharacter404"
    >
      <div
        class="words"
      >
        <div
          class="word"
        >
          &lt;?php
        </div>
        <div
          class="word"
        >
          &lt;div&gt;
        </div>
      </div>
      <div
        class="words"
      >
        <div
          class="word-right"
        >
          &lt;html&gt;
        </div>
        <div
          class="word-right"
        >
          body
        </div>
      </div>
      <div
        class="chair404"
      />
      <div
        class="leftshoe404"
      />
      <div
        class="rightshoe404"
      />
      <div
        class="legs404"
      />
      <div
        class="torso404"
      >
        <div
          class="body404"
        />
        <div
          class="leftarm404"
        />
        <div
          class="rightarm404"
        />
        <div
          class="head404"
        >
          <div
            class="eyes404"
          />
        </div>
      </div>
      <div
        class="laptop404"
      />
    </div>
  </div>
</div>
`;
